build/compile_descriptor

```diff
@@ -45,12 +45,13 @@ main(_) ->
         case REBAR_DEPS_DIR =/= false
             andalso filelib:is_dir(filename:join(REBAR_DEPS_DIR, "gpb")) of
             true ->
-                Ebin = filename:join([REBAR_DEPS_DIR, "gpb", "ebin"]),
-                filelib:ensure_dir(filename:join(Ebin, ".dummy")),
-                {Ebin, filename:join([REBAR_DEPS_DIR, "gpb", "test"])};
+                {filename:join([REBAR_DEPS_DIR, "gpb", "ebin"]),
+                 filename:join([REBAR_DEPS_DIR, "gpb", "test"])};
```

Comment:
please correct me if I'm wrong here about the best way to format this little block: I have no idea what I'm doing with erlang formatting :sweat_smile: